File: Yijing.swt/Yijing/src/org/yijing/CBitValueSequencer.java
package org.yijing;

public class CBitValueSequencer extends CValueSequencer
{
	public CBitValueSequencer(int nValue) {
		super(0, 2, nValue);
		m_nSequences = Sequences.nBitSequences;
		m_nRatios = Sequences.nBitRatios;
	}
	@Override
	public void UpdateOuterValues() {
		m_pvsParent.UpdateOuterValues();
	}
}

File: ValueSequencer.ts/CBitValueSequencer.ts
import { CValueSequencer } from "./CValueSequencer.ts";
import { Sequences } from "./Sequences.ts";

export class CBitValueSequencer extends CValueSequencer {
  constructor(nValue: number) {
    super(0, 2, nValue);
    this.m_nSequences = Sequences.nBitSequences;
    this.m_nRatios = Sequences.nBitRatios;
  }

  public override updateOuterValues(): void {
    this.m_pvsParent?.updateOuterValues();
  }
}
